README.md

```diff
@@ -78,6 +78,7 @@ This is the process that would typically be deployed as a DaemonSet to ensure th
 | [calico/canal](https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation/flannel) | `cali+` |   |
 | [kube-router](https://www.kube-router.io/docs) | `kube-bridge` | This is the default bridge interface that all the pods are connected to when using kube-router |
 | [kops (kubenet)]( https://github.com/kubernetes/kops/blob/master/docs/networking.md) | `cbr0` | When using the default CNI (kubenet) |
+| [celium](https://docs.cilium.io/) | `lxc+` |  |
```

Comment:
thanks for the PR, `celium` should be `cilium`

Reply:
Sorry about the typo, fixed it!

Reply:
no probs :)